docs/config.md

```diff
@@ -87,6 +87,13 @@ nodes:
 $ kubectl aks use-node aks-agentpool-12345678-vmss000000
 ```
 
+Or in case Kubernetes API is not available, we can also import the node information using Azure API:
```

Comment:
```suggestion
Or in case Kubernetes API is not available, we can also import the node information using Azure API by specifying the cluster information:
```